zerver/tests/test_upload.py

```diff
@@ -59,8 +59,8 @@ def test_rest_endpoint(self) -> None:
         # Upload file via API
         result = self.api_post(self.example_user("hamlet"), "/api/v1/user_uploads", {"file": fp})
         response_dict = self.assert_json_success(result)
-        self.assertIn("uri", response_dict)
-        url = response_dict["uri"]
+        self.assertIn("url", response_dict)
+        url = response_dict["url"]
         base = "/user_uploads/"
```

Comment:
Should I keep checks for `uri`?

Reply:
Yes, we should always test the backwards-compatible code when leaving it in place.

Reply:
OK, I have updated the PR. Can you please review that?